headers/functions/kaomado.h

```diff
@@ -0,0 +1,11 @@
+// All Kaomado loads use the same stream, so it's initialized once on game start!
+void InitStreamKaomado(void);

```

Comment:
Can you put these in `arm9.h`? Making new function files will take a bit more work:
1. Putting `#include` in the right places so that it's reachable from `pmdsky.h`.
2. Modifying `symbol_check.py` to be more robust so it can cross-check different files against a single symbol file.

As I mentioned, I think it's a bit early to be doing this kind of micro-organization.